integration/single-node-slug-deploy/check.go

```diff
@@ -62,6 +163,29 @@ func main() {
 	}
 }
 
+func loadGPG(workdir string) error {
+	importpub := exec.Command("gpg", "--no-default-keyring", "--keyring", path.Join(workdir, "pubring.gpg"), "--secret-keyring", path.Join(workdir, "secring.gpg"), "--import")
```

Comment:
Why not just create the pubring upfront and save it in this test directory? You can use `util.From` to make it a little easier to locate.